packages/templates/api/assemblyscript/yarn.lock

```diff
@@ -0,0 +1,5888 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
yarn.lock files shouldn't be versioned for packages, and also shouldn't be present in template projects as a whole. They should be left to be generated client-side when users create the template project.